src/content/en/updates/2019/08/barcode-detection.md

```diff
@@ -0,0 +1,140 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Barcode Detection API allows for barcodes to be detected in images.
+
+{# wf_updated_on: 2019-08-06 #}
+{# wf_published_on: 2019-08-01 #}
+{# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: The Barcode Detection API allows for barcodes to be detected in images. #}
+{# wf_blink_components: Blink>ImageCapture #}
```

Comment:
Blink>ShapeDetection (this also needs to be fixed in `shape-detection.md` but I cannot comment on lines that have not changed.

Reply:
Fixed, overlooked the front matter, thanks for the eagle eye!